src/test/java/io/aiven/kafka/connect/common/output/JsonLinesOutputWriterTest.java

```diff
@@ -80,6 +79,22 @@ void jsonValueWithValue() throws IOException {
         assertRecords(Collections.singletonList(record1), expected);
     }
 
+    @Test
+    void jsonValueWithSingleField() throws IOException {
```

Comment:
It's worth checking with the enabled envelope as well

Reply:
It is enabled by default, so we have the tests covering that case. Or should we have a dedicated one?

Reply:
Right. Yeah, no worries then